src/unwrap.jl

```diff
@@ -263,16 +263,22 @@ function calculate_reliability(pixel_image::AbstractArray{T, N}, circular_dims,
         @inbounds pixel_image[i].reliability = calculate_pixel_reliability(pixel_image, i, pixel_shifts, range)
     end
 
+    if !(true in circular_dims)
+        return
+    end
+
+    pixel_shifts_border = similar(pixel_shifts)
+    new_ps = zeros(N)
```

Comment:
```suggestion
    new_ps = zeros(Int, N)
```
Might be the cause of the CI failures?